tests/update-ci-snapshots/Dockerfile

```diff
@@ -4,6 +4,11 @@
 FROM mcr.microsoft.com/playwright:v1.30.0-focal
 WORKDIR /work/
 
+# Copy the web-client-ui src folder to the docker image
+# This requires the Dockerfile to be built in the context of the root of the web-client-ui repository
+# https://stackoverflow.com/a/34300129
+ADD . /work
```

Comment:
I think we should be using `COPY` here https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Also, we may want to copy just a few select files initially after the nvm install (like .nvmrc, package.json files, package-lock.json), then `RUN npm ci` and then copy the remaining files (ideally ignoring dist/build related files). This should improve docker caching these steps

I can see why some repos build to a `/build` folder in the root instead of within the individual packages. Makes copy/cache easier in docker since the `packages` directory should only contain src files at any state locally

Although cacheability shouldn't matter as much if the plan is to tell people to run locally and only run this before they push since it will only take a few extra minutes.

In theory though, I think w/ those steps that this should be similar in speed to just running locally. Would need to rebuild if the src files changed, but that's it.

Reply:
Yeah I don't know how to copy while ignoring the dist/build files. No clean way to do that (that I can tell) unless we change the build output folder to not be in package directories at all.

Reply:
[.dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file) will do just that actually. A bit annoying that it doesn't just read gitignore, but we could just throw it in a directory w/ the dockerfile inside test or something